### PR TITLE
Add [Gen 7] Metronome Battle ladder

### PR DESCRIPTION
**Pull request by Ridaz**

### CODE REVIEW

File: config/formats.js [hunk 1]
```diff
@@ -707,6 +674,37 @@ let Formats = [
 		team: 'randomHC',
 		searchShow: false,
 		ruleset: ['Pokemon', 'HP Percentage Mod', 'Cancel Mod'],
+	},	{
```
Comment (AustinXII) [inline config/formats.js]: ?

Comment (QuiteQuiet) [reply config/formats.js]: what austin means is that all the other are like 
```javascript
},
{
```

**Comment by TheImmortal** [comment]
I’m not sure about classifying it as a Random meta. As I said, I prefer it under Doubles.

@zarel thoughts?

File: config/formats.js (hunk 1 above)
Comment (AustinXII) [reply config/formats.js]: Sorry I assumed that was incredibly obvious

**Comment by Ridaz** [comment]
The argument here is, since Metronome's the decisive random factor, should or should it not go under Doubles or Randomized formats

**Comment by KrisXV** [comment]
Unless the Pokémon are random too, there’s no reason for it to be a randomized meta

**Comment by Zarel** [comment]
I would not call it a random meta if it requires teambuilding.

**Comment by Ridaz** [comment]
Fine, I'll move it under Doubles

**Comment by Ridaz** [comment]
Do not merge it yet, still need to ban assault vest

File: config/formats.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
 let Formats = [
 
 	// US/UM Singles
-	///////////////////////////////////////////////////////////////////
+ 	///////////////////////////////////////////////////////////////////
```
Comment (TheImmortal) [inline config/formats.js]: ?

**Comment by TheImmortal** [comment]
I’m pretty sure it’s still under Randoms

**Comment by Ridaz** [comment]
Apparently it decided to revert the commit where I moved the format

**Pull request merged**